migrations/001_containers_basic_meta.sql

```diff
@@ -82,10 +82,6 @@ CREATE TABLE metatype_relationship_keys (
     modified_by character varying(255) NOT NULL
 );
 
-
```

Comment:
I can't get these unique constraints to drop any other way than not adding them in the first place. We already edited previous sql scripts once, lets go ahead and accept it again today if that's ok